src/actions/creators.js

```diff
@@ -0,0 +1,9 @@
+import { NEW_DATA_LOADED, DRAWER_TOGGLED, NAME_CHANGED, SURNAME_CHANGED, MAIL_CHANGED, PASSWORD_CHANGED } from "./types";
+
+const newDataLoaded = (key, content) => ({ type: NEW_DATA_LOADED, content: content, keyValue: key });
+const drawerToggled = (open) => ({ type: DRAWER_TOGGLED, value: open });
+const nameChanged = (name) => ({ type: NAME_CHANGED, value: name });
```

Comment:
Putting form data to redux is too much. It's almost impossible that this data will be shared with any other component so far away that it makes sense to add it to Redux.